### PR TITLE
Add client id to thirdweb template

### PR DESCRIPTION
**Pull request by warengonzaga**
Hey @migueloller, added the `client id` to the template to follow our latest changes. We now require everyone to use Client ID if you use SDK's client side. More info here: https://blog.thirdweb.com/changelog/api-keys-to-access-thirdweb-infra

### CODE REVIEW

**Comment by warengonzaga** [comment]
Before this update
image

After this update
image

@migueloller

File: examples/thirdweb/components/NFTDropCard/NFTDropCard.tsx [hunk 1]
```diff
@@ -59,6 +59,7 @@ type Props = {
   className?: string
   contractAddress?: string
   chainId?: string
+  clientId?: string
```
Comment (migueloller) [inline examples/thirdweb/components/NFTDropCard/NFTDropCard.tsx:62]: This `clientId` prop doesn't seem to be used by the `NFTDropCard` component

**Review by migueloller — CHANGES_REQUESTED** [review]
Looks simple enough. Do you mind squashing the commits and also removing the unused `clientId` from the `NFTDropCard` component?

File: examples/thirdweb/components/NFTDropCard/NFTDropCard.tsx (hunk 1 above)
Comment (warengonzaga) [reply examples/thirdweb/components/NFTDropCard/NFTDropCard.tsx:62]: Let me take a look @migueloller

Comment (warengonzaga) [reply examples/thirdweb/components/NFTDropCard/NFTDropCard.tsx]: Hey @migueloller removed the unused `clientId` from the `NFTDropCard` component.

**Review by migueloller — APPROVED** [review]

**Pull request merged**